scipy/interpolate/_cubic.py

```diff
@@ -556,7 +566,9 @@ def __init__(self, x, y, axis=0, *, method: Literal["akima", "makima"]="akima",
             f12 = f1 + f2
 
             # These are the mask of where the slope at breakpoint is defined:
-            ind = np.nonzero(f12 > break_mult * np.max(f12, initial=-np.inf))
+            mmax = xp.asarray(np.max(np.asarray(f12), initial=-np.inf))
+            ind = xp.nonzero(f12 > break_mult * mmax)
```

Comment:
Possibly?
```suggestion
            mmax = xp.max(f12) if f12.size > 0 else -xp.inf
            ind = xp.nonzero(f12 > break_mult * mmax)
```

Reply:
Thanks, done. This `initial=` thingy is not a little esoteric indeed.